app/assets/javascripts/components/students/shared/StudentList/StudentList.jsx

```diff
@@ -78,12 +95,7 @@ export const StudentList = (props) => {
       sortable: true,
       info_key: 'users.revisions_doc'
     },
-    character_sum_ms: {
-      label: I18n.t('users.chars_added'),
-      desktop_only: true,
-      sortable: true,
-      info_key: 'users.character_doc'
-    },
+    character_sum_ms: contentAddedKey,
```

Comment:
I think this ends up being a bit confusing given that all of these declarations are all over the case. Maybe it makes sense to make a separate `.js` file that just includes the keys, or to somehow otherwise "chunk" the relevant key code together.